packages/editor/src/components/collab-sidebar/comment-form.js

```diff
@@ -65,6 +79,9 @@ function CommentForm( {
 				}
 				rows={ 1 }
 				maxRows={ 20 }
+				ref={
+					shouldFocusTextareaOnMount ? focusOnMountRef : textareaRef
+				}
```

Comment:
```suggestion
				ref={
					shouldFocusTextareaOnMount ? focusOnMountRef : null
				}
```
What is textareaRef used for?

Reply:
Updated